src/lake/examples/reverse-ffi/main.c

```diff
@@ -3,7 +3,7 @@
 
 extern uint64_t my_length(lean_obj_arg);
 
-// see https://leanprover.github.io/lean4/doc/dev/ffi.html#initialization
+// see https://lean-lang.org/doc/reference/latest/find/?domain=Verso.Genre.Manual.section&name=ffi-initialization
```

Comment:
This link is currently broken. It produces a "Not found: name 'ffi-initialization'" page on the manual.

Reply:
This will be fixed as soon as the manual for the RC comes out. Let's wait to merge until that happens.

Reply:
The link now works.